src/applications/disability-benefits/2346/components/IntroductionPage.jsx

```diff
@@ -30,26 +29,33 @@ class IntroductionPage extends React.Component {
         <div className="process schemaform-process">
           <ol>
             <li className="process-step list-one">
-              <h3>Prepare</h3>
-              <p>To place an order, you'll need your:</p>
+              <h3 className="vads-u-font-size--h5">Prepare</h3>
```

Comment:
All these header sizes look way different from the work I am doing on the HLR form. I'm not sure which is right?
<details><summary>Screenshot</summary>

<!-- leave a blank line above -->
![Screen Shot 2020-04-09 at 4 14 40 PM](https://user-images.githubusercontent.com/136959/78941485-66a7e000-7a7d-11ea-8c01-c3a0a77cc3e1.png)
</details>

Reply:
@Mottie yes this was wrong - fixed